bin/node/src/main.rs

```diff
@@ -122,3 +122,41 @@ async fn main() -> anyhow::Result<()> {
         },
     }
 }
+
+/// Generates the `--version` output enriched with additional build metadata.
+fn long_version() -> String {
```

Comment:
Could this function (or most of it) go into the utils crate? As far as I can tell, it mostly just looks up environment variables and joins them together in a formatted string.

Reply:
Oh that's a great point. I was dreading copy-pasting this around and completely lost sight of the fact that its just the `build.rs` that needs to be recompiled.